terraform/globals.tf

```diff
@@ -1,3 +1,7 @@
+terraform {
+  required_version = "0.8.7"
+}
```

Comment:
This isn't the correct place to add this - the pingdom terraform doesn't include this file. There's also a separate `version_constraint.tf` file that's included in all the terraform configs that already specifies the version, although it's currently more relaxed and only specifies `>= 0.8.5`.

I've thought that we should pin the version we're using globally more strictly - we should make it exactly match the version in `paas-docker-cloudfoundry-tools`, but that's currently 0.8.5, which poses problems for terraform configs that have moved beyond that.

From here, I think there are 2 options. We can either pin the version used by the pingdom config separately from the rest (which isn't great for consistency), or we can defer the changes to the version pinning to a later story, and reduce this PR to the second commit only (to save unnecessary downloads).